unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/States/EditorMode/BuilderInWorldGodMode.cs

```diff
@@ -359,19 +360,19 @@ public override void Activate(ParcelScene scene)
         Environment.i.world.sceneController.IsolateScene(sceneToEdit);
         Utils.UnlockCursor();
 
-        RenderSettings.fog = false;
         gizmoManager.HideGizmo();
         editionGO.transform.SetParent(null);
         avatarRenderer.SetAvatarVisibility(false);
+
+        if (HUDController.i.builderInWorldMainHud != null)
+            HUDController.i.builderInWorldMainHud.ActivateGodModeUI();
```

Comment:
You can use the null-conditial operator here:
```suggestion
HUDController.i.builderInWorldMainHud?.ActivateGodModeUI();
```